src/Enum.php

```diff
@@ -117,7 +117,7 @@ public function __toString(): string
      *
      * @return string
      */
-    final public function getName(): string
+    final public function getName()
```

Comment:
That belongs to the public API and should keep type hints.

Reply:
This method is marked final. So there is no difference for end users for return type hint vs. phpdoc as long as we are 100% sure the expected type gets returned.

Reply:
Btw. in this simple example  it's 4.936μs vs. 4.672μs - which is 5.35%.